### ISSUE TITLE
Missing .pre-commit-hooks.yaml in 1.1.3

### ISSUE THREAD

**Issue by torfsen** [econchick/interrogate#10]
**Is this a BUG REPORT or FEATURE REQUEST?**:

* [x] bug
* [ ] feature

## Description of Bug or Feature

The docs currently [contain](https://interrogate.readthedocs.io/en/latest/#other-usage) the following config for using interrogate with pre-commit:
```yaml
repos:
  - repo: https://github.com/econchick/interrogate
    rev: 1.1.3
    hooks:
      - id: interrogate
        args: [--quiet, --fail-under=95]
```
However, that fails for me with Python 3.7 and pre-commit 2.3.0 with the error message
```
/home/my-user/.cache/pre-commit/repo59ixepcf/.pre-commit-hooks.yaml does not exist
```
And indeed, the file seems not be in the repo for the [1.1.3 tag](https://github.com/econchick/interrogate/tree/1.1.3). It is in `master`, so setting the `rev` to `master` is a temporary workaround. Obviously it would be desirable to pin a fixed version in the long run.

**Comment by econchick** [comment]
shoot, good catch! Give me a few minutes and I'll make a patch release. It should then work for `1.1.4`. Thanks so much for leading me know!

**Comment by torfsen** [comment]
I can confirm that this is fixed in 1.1.4. Thanks for the quick response!
